Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -1,20 +1,40 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultEditRequest {
+public class BTPayPalVaultEditRequest {
 
+    var riskCorrelationID: String?
+    let hermesPath: String = "v1/paypal_hermes/generate_edit_fi_url"
     private let editPayPalVaultID: String
-    private let merchantAccountID: String?
```

Comment:
Why is this being removed? How will merchants use their non default merchantAccountID?

Reply:
This was to be used in second endpoint that is now being moved to server side, so we no longer need it. Since we are passing in encrypted billing agreement ID, I think we don't need it.
This endpoint never took that as a param to Gateway function.